crates/nu-command/tests/commands/into_string.rs

```diff
@@ -0,0 +1,49 @@
+use nu_test_support::{nu, pipeline};
```

Comment:
We don't really need this file, these testing code can be moved to `crates/nu-command/tests/commands/str_/into_string.rs`